airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -255,11 +255,16 @@ def execute(self, context: Context):
 
         return self.job_id
 
-    def execute_complete(self, context, event=None):
+    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> str:
```

Comment:
I see the execute_complete function mostly same across multiple aws operators (batch, ear, glue, crawler and etc). It's worth to re consider it refactoring to avoid code duplication.

Reply:
Yep, might need to take a deeper look at what are the same and what are the differences. Thanks for your suggestion!

Reply:
I tried to extract a utility function and check the event instead. The detail and even signature are different between  execute_complete methods